examples/pipelines/commoncrawl/components/download_commoncrawl_segments/src/utils/text_utils.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from bs4 import BeautifulSoup
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = "https://data.commoncrawl.org/"
+
+
+def convert_to_plain_text(html: str) -> str:
+    """Converts an HTML string to plain text.
+    Args:
+        html: The HTML string.
+    Returns:
+        The plain text string.
+    """
+    try:
+        soup = BeautifulSoup(html, "html.parser")
+        return soup.get_text()
```

Comment:
Currently there seems to be a performance bottleneck by converting the page into plain text.
I have found a different library (specially designed for this use case) that seems to be faster and more robust.
Can you check out trafilatura? https://trafilatura.readthedocs.io/en/latest/

Reply:
done. I'll update the PR to reflect the change.